takeshape/penny
# Use NextAuth and custom OIDC logic

## Test Plan

1. Sign in by clicking the nav icon
2. Go to `/purchases` and `/account`, everything working?
3. Logout from the `/account` page
4. Attempt to checkout when signed out, you should be prompted to sign in, and your checkout session should continue.
